test/e2e/test-plugins/plugins-api/sidebar.js

```diff
@@ -5,6 +5,8 @@
 	var compose = wp.compose.compose;
```

Comment:
We should probably add `wp-annotations` as a dependency for the WP script targetting this file.

Reply:
Done.